pyproject.toml

```diff
@@ -16,7 +16,8 @@ classifiers = [
 ]
 dependencies = [
   "pandas",
-  "btrack"
+  "btrack",
+  "click"
```

Comment:
Comment that I also wanted to make in #12: I think it'd be better to specify compatibilities, like I did in #2 at https://github.com/Jamie-Dean-Lab/Lineage-analysis/blob/7f21b6285a965fd8234714fc8a3f44f94c36b2d7/pyproject.toml#L21

Side note, always leaving a trailing comma in these lists allows you not to modify the previous line when you want to add a new item, which keeps git history (and git blame in particular) cleaner :wink:

Reply:
Thanks @giordano ! I've pinned all the main dependencies to `>=latest_version <next_major_version` (except for the dev dependencies, as these are optional and only needed for testing/linting). I also added a trailing comma :smile: